### PR TITLE
Aeif psc delta 2 continued

### PR DESCRIPTION
**Pull request by stinebuu**
This is a continuation of PR #613, and can replace that PR. Most of the work has been reviewed, the difference is that `with_refr_input_` is defaulted to `false` and is made set/getable as requested. I have also reordered the placement of `with_refr_input_` in `aeif_psc_delta.h` to get rid of a warning.

Almost all the work is done @lepmik.

### CODE REVIEW

File: pynest/nest/tests/test_aeif_lsodar.py [hunk 1]
```diff
@@ -249,10 +268,10 @@ def assert_pass_tolerance(self, rel_diff, di_tol):
                                     model, var, diff, di_tol[model][var]))
 
     @unittest.skipIf(not HAVE_GSL, 'GSL is not available')
-    def testClosenessNestLSODAR(self):
-
-        # Compare models to the LSODAR implementation.
-
+    def test_closeness_nest_lsodar(self):
```
Comment (Silmathoron) [inline pynest/nest/tests/test_aeif_lsodar.py]: this was changed by @gtrensch is it okay to switch back to docstrings now?

File: pynest/nest/tests/test_aeif_lsodar.py [hunk 2]
```diff
@@ -134,6 +138,17 @@
     'tau_syn_ex': 0.2
 }
 
+aeif_DT0_delta = aeif_DT0.copy()
+aeif_DT0_delta.pop('tau_syn_ex')
+
+DT0_params = {"aeif_cond_alpha": aeif_DT0,
```
Comment (Silmathoron) [inline pynest/nest/tests/test_aeif_lsodar.py]: use [``defaultdict``](https://docs.python.org/2.7/library/collections.html#collections.defaultdict)?

**Comment by stinebuu** [comment]
Sorry I took so long to address your comments, @Silmathoron!

**Comment by heplesser** [comment]
@lepmik @stinebuu Thanks for your efforts, merging!

**Pull request merged**